src/ceph-volume/ceph_volume/devices/lvm/zap.py

```diff
@@ -66,6 +66,7 @@ def zap_data(path):
         'of={path}'.format(path=path),
         'bs=1M',
         'count=10',
+        'conv=fsync'
```

Comment:
I wonder if you could/should add 'oflag=direct' as well.

Reply:
I don't think its needed. We want to ensure the data is written to disk before dd returns, using direct I/O is a stricter requirement.

In any case this is a backport PR, so this would be the wrong place anyway ;)